client/src/components/NewEvent.js

```diff
@@ -0,0 +1,68 @@
+import {useState} from 'react';
+
+function NewEvent () {
+    const [title, setTitle] = useState('')
+    const [isVirtual, setVirtual] = useState(false)
+    const [location, setLocation] = useState('')
+    const [description, setDescription] = useState('')
+    const [startDate, setDate] = useState('')
+    const [endDate, setEnd] = useState('');
+
+    function eventSubmit (e) {
+        e.preventDefault();
+        let obj = {
+            title,
+            isVirtual,
+            location,
+            description,
+            time_start: new Date(startDate),
+            time_end: new Date(endDate)
+        }
+
+        fetch('http://localhost:5000/events/new', {
+            method: 'POST',
+            headers: {'Content-Type': 'application/json'},
+            body: JSON.stringify(obj)
+        })
+
+        setTitle('');
+        setVirtual(false);
+        setLocation('');
+        setDescription('');
+        setDate('');
+        setEnd('');
```

Comment:
Wondering about how useful resetting the state will be since we would probably redirect the user to another page... 🤔